src/machine/machine.c

```diff
@@ -746,6 +746,214 @@ int machine_get_uid_shift(Machine *m, uid_t *ret) {
         return 0;
 }
 
+int machine_owns_uid(Machine *machine, uid_t uid, uid_t *ret_internal_uid) {
+        char p[STRLEN("/proc//uid_map") + DECIMAL_STR_MAX(pid_t) + 1];
```

Comment:
Why do you need here `//` in `/proc//uid_map` ?

Reply:
The number goes between those slashes.